### PR TITLE
pages.pt_BR/linux/*: add Portuguese Brazilian translations

### PR DESCRIPTION
**Pull request by rffontenelle**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.pt_BR/linux/pacgraph.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+`pacgraph --dep={{cor}}`
+
+- Altera a cor de fundo de um grafo:
+
+`pacgraph --background={{cor}}`
+
+- Altera a cor dos links entre pacotes:
+
+`pacgraph --link={{color}}`
```
Comment (renie) [inline pages.pt_BR/linux/pacgraph.md:36]:
```suggestion
`pacgraph --link={{cor}}`
```

**Review by renie — CHANGES_REQUESTED** [review]
Just a small change. You probably just missed that word =)

File: pages.pt_BR/linux/pacgraph.md (hunk 1 above)
Comment (rffontenelle) [reply pages.pt_BR/linux/pacgraph.md:36]: ops

**Review by renie — APPROVED** [review]
All good now. Thanks for one more contribution, @rffontenelle ! Vai Brasil :beach_umbrella: !

**Pull request merged**